source/fundamentals/logging.txt

```diff
@@ -0,0 +1,273 @@
+=======
+Logging
+=======
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. _logging:
+
+Overview
+--------
+
+.. What is on this page?
+
+In this guide, we show you how to set up and configure a logger in the
+MongoDB Java driver.
+
+.. What do any new terms mean?
+
+.. Not applicable. We are treating definition of logger as understood.
+
+.. What can you expect to see on this page?
+
+In this guide you can learn how to do the following:
+
+- Set up a logger using the Simple Logging Facade For Java (SLF4J)
+- Configure the log level of your logger
+
+.. Who should read this? Where should I go if this isn't the page I was looking for?
+
+.. Not Applicable. No natural redirect location.
+
+Set Up a Logger
+---------------
+
+To set up a logger for your project, you need to add some dependencies.
+
+The MongoDB Java driver uses the Simple Logging Facade For Java (SLF4J).
+SLF4J allows you to specify your logging framework of choice at deployment time.
+For more information on SLF4J,
+`see the SLF4J documentation <http://www.slf4j.org/>`__.
+
+To set up a logger you must add the ``slf4j-api`` package, a logging framework, and a
+**binding** to your project's dependency list. A binding is a piece of code that
+connects the ``slf4j-api`` package with a logging framework. The two most
+popular logging frameworks used with ``slf4j-api`` are Log4j2 and Logback.
+
+Setting up a logger is optional. When you start your application the MongoDB
+Java driver looks for the ``slf4j-api`` package in your classpath. If the driver
+can't find ``slf4j-api``, the driver logs a single warning with
+``java.util.logging`` and disables all further logging.
+
+Here is the error the driver outputs if it cannot find the ``slf4j-api`` package:
+
+.. code-block:: none
+
+   WARNING: SLF4J not found on the classpath.  Logging is disabled for the 'org.mongodb.driver' component
+
```

Comment:
nitpick: Convert to an active action. Also, is it an error or warning?

> The driver will output a single warning when no logging framework is configured:

Reply:
Thank you for the note. I agree that phrasing this in terms of the action the driver is performing improves the sentence. Also it is a warning, good catch. Apologies for mixing up my terms. Fix:

Going forward, I will try and phrase things in terms of actions being performed rather than things that exist.